src/components/PlaceholderContainer/PlaceholderContainer.scss

```diff
@@ -0,0 +1,136 @@
+@use '../variables';
+@use '@gravity-ui/uikit/styles/mixins';
+
+$htmlMargin: 40px;
+
+$imageSmallSize: 100px;
+$imageMediumSize: 150px;
+$imageLargeSize: 230px;
+$imagePromoSize: 200px;
+
+$block: '.#{variables.$ns}placeholder-container';
+
+#{$block} {
+    $class: &;
```

Comment:
$class is the same as $block, no need to have it

Reply:
fixed